open_schools_platform/common/firebase/utils.py

```diff
@@ -0,0 +1,15 @@
+from firebase_admin import messaging
+
+from open_schools_platform.common.firebase.settings import app
+from open_schools_platform.user_management.users.models import User
+
+
+def notify_user(user: User, title: str, body: str, data: dict = None):
+    if user.firebase_token.token is not None:
+        message = messaging.Message(
+            android=messaging.AndroidConfig(
+                notification=messaging.AndroidNotification(title=title, body=body)),
+            data=data,
+            token=user.firebase_token.token,
+        )
+        messaging.send(message, app=app)
```

Comment:
let this funcrtion return some value, for example:
0 - notification send succesfully
1 - firbase_token is None
2 - error when sending
make docstring for this responses

or you can return response
#response = messaging.send(message, app=app)

Reply:
Do you mean returning 0, 1 and 2 as numbers? Can we just raise exceptions if something is wrong?

Reply:
I think numeric return codes are nice solution for this problems